bbb-graphql-actions/src/actions/presentationRequestUploadToken.ts

```diff
@@ -13,6 +14,12 @@ export default function buildRedisMessage(sessionVariables: Record<string, unkno
 
   const eventName = `PresentationUploadTokenReqMsg`;
 
+  const filename: string = input.filename as string;
+  const uploadTemporaryId: string = input.uploadTemporaryId as string;
+  if(uploadTemporaryId.length > 500 || filename.length > 500) {
+    throw new ValidationError('Presentation name or temporaryId with more than 500 characters', 400);
+  }
+
```

Comment:
I suggest to follow the pattern of the errors messages used in `validation.ts`.
So you could validate each parameter individually and return something like:

```js
throw new ValidationError(`Parameter '${param.name}' exceeds maximum allowed length of 500 characters.`, 400);
```